lib/controllers/cryptids.js

```diff
@@ -0,0 +1,35 @@
+const { Router } = require('express');
+const { Cryptid } = require('../models/cryptids');
+
+module.exports = Router()
+  .get('/:id', async (req, res, next) => {
+    try {
+      const cryptid = await Cryptid.getById(req.params.id);
+      if (!cryptid) {
+        next();
+      }
+      res.json(cryptid);
+    } catch (e) {
+      next(e);
+    }
+  })
+  .get('/', async (req, res) => {
```

Comment:
we should be doing ```try catch``` on all of these — not on the rubric but you wanna handle your errors so when something goes wrong you have a little direction